sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 # Release History
 
-## 3.3.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 3.3.2 (2023-11-07)
 
 ### Bugs Fixed
 - Fixed incorrect data type for returned formula objects.
 
-### Other Changes
```

Comment:
Should the fixes for send request have been added in the changelog?

Reply:
No. There's no behavior changes for customers, the fixe is like an optimize for v2 api versions.